Content.Shared/Actions/ActionTypes/ActionType.cs

```diff
@@ -237,6 +237,7 @@ public virtual void CopyFrom(object objectToClone)
         AutoPopulate = toClone.AutoPopulate;
         AutoRemove = toClone.AutoRemove;
         ItemIconStyle = toClone.ItemIconStyle;
+        ClientExclusive = toClone.ClientExclusive;
         CheckCanInteract = toClone.CheckCanInteract;
         UseDelay = toClone.UseDelay;
         Sound = toClone.Sound;
```

Comment:
this was causing the clientside actions to get removed after dirty since it wasnt cloning `ClientExclusive` properly